source/PlayerInfo.cpp

```diff
@@ -875,6 +840,53 @@ Account &PlayerInfo::Accounts()
 
 
 
+// Handle the daily salaries and payments.
+void PlayerInfo::DoAccounting()
+{
+	// Check what salaries and tribute the player receives.
+	int64_t salariesIncome = accounts.SalariesIncomeTotal();
+	int64_t tributeIncome = GetTributeTotal();
+	FleetBalance b = MaintenanceAndReturns();
+	if(salariesIncome || tributeIncome || b.assetsReturns)
+	{
+		string message = "You receive ";
+		if(salariesIncome)
+			message += Format::CreditString(salariesIncome) + " salary";
+		if(salariesIncome && tributeIncome)
+		{
+			if(b.assetsReturns)
+				message += ", ";
+			else
+				message += " and ";
+		}
```

Comment:
```suggestion
		if(salariesIncome)
		{
			message += Format::CreditString(salariesIncome) + " salary";
			if(tributeIncome)
			{
				if(b.assetsReturns)
					message += ", ";
				else
					message += " and ";
			}
		}
```
dunno, I think this looks better

Reply:
Again, this is the original code, but this is definitely an improvement.